src/pages/workspace/categories/WorkspaceCategoriesSettingsPage.tsx

```diff
@@ -45,20 +73,33 @@ function WorkspaceCategoriesSettingsPage({policy, route}: WorkspaceCategoriesSet
                 testID={WorkspaceCategoriesSettingsPage.displayName}
             >
                 <HeaderWithBackButton title={translate('common.settings')} />
-                <View style={styles.flexGrow1}>
-                    <ToggleSettingOptionRow
-                        title={translate('workspace.categories.requiresCategory')}
-                        subtitle={toggleSubtitle}
-                        switchAccessibilityLabel={translate('workspace.categories.requiresCategory')}
-                        isActive={policy?.requiresCategory ?? false}
-                        onToggle={updateWorkspaceRequiresCategory}
-                        pendingAction={policy?.pendingFields?.requiresCategory}
-                        disabled={!policy?.areCategoriesEnabled || !hasEnabledOptions || isConnectedToAccounting}
-                        wrapperStyle={[styles.mt2, styles.mh4]}
-                        errors={policy?.errorFields?.requiresCategory ?? undefined}
-                        onCloseError={() => Policy.clearPolicyErrorField(policy?.id ?? '-1', 'requiresCategory')}
-                        shouldPlaceSubtitleBelowSwitch
-                    />
+                <ToggleSettingOptionRow
+                    title={translate('workspace.categories.requiresCategory')}
+                    subtitle={toggleSubtitle}
+                    switchAccessibilityLabel={translate('workspace.categories.requiresCategory')}
+                    isActive={policy?.requiresCategory ?? false}
```

Comment:
isn't it defaulting to false?

Reply:
`isActive` is a required prop by `ToggleSettingOptionRow` so we can't return `undefined` (which `policy?.requiresCategory` can return). I believe the author didn't want to change types of `ToggleSettingOptionRow` component